webapp/src/main/webapp/index.htm

```diff
@@ -38,7 +38,11 @@
         	<td width="100px"><div class="logo"></div></td>
         	<td>
             	<div class="bar-round">
-        			<div class="bar">OpenMRS Platform ${project.version}.${revisionNumber} Running!</div>
+        			<div class="bar">
+        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+        				<p>If you seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. To learn about the available user interfaces, see [<a href="https://wiki.openmrs.org/display/docs/Administering+Modules">https://wiki.openmrs.org/display/docs/Administering+Module/</a>]</p>
```

Comment:
"Administering Modules" is the wrong link here. We should create a _new_ wiki page called something like "User Interface Modules" which should describe that the OpenMRS Platform does not have a UI starting from version 2.0, and that they can install either (a) Legacy UI module, (b) Reference Application, (c) something else...